inc/driver.h

```diff
@@ -25,9 +25,9 @@
  * 100% / 250 counts == 0.4% per count
  * 250 * .004 == 1
  */
-#define PWM_PERCENT_PER_COUNT_250  0.004 //  ( 100.0 / MDATA_TBL_SIZE ) // .4% i.e. .004
+#define PWM_PERCENT_PER_COUNT_250  0.004 //  ( 100.0 / MDATA_TBL_SIZE ) // .4% 
```

Comment:
To be clear, the open-loop look up table is long gone, but leaves its legacy, that being the UI commands the % speed (PWM DC) in 250 steps, whereas the radio signal will have 10-bit resolution so 1024 steps.